src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationTest.java

```diff
@@ -33,6 +33,7 @@ public void find_users_by_active_organisation_with_showDeleted_False() {
         List<HashMap> professionalUsersResponses = (List<HashMap>) searchResponse.get("users");
         HashMap professionalUsersResponse = professionalUsersResponses.get(0);
 
+        assertThat(professionalUsersResponse.get("userIdentifier")).isNotNull();

```

Comment:
Ok, but this code on line 33 at the top of the method body needs to be refactored so that there is no duplicate code. You can put the creation of the response in a setUp method with @Before and assign a private member variable instead.

Reply:
Refactored.